ajax/libs/instagram-lite/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "instagram-lite",
+  "version": "2.0.0",
+  "filename": "instagramLite.min.js",
+  "description": "A simple, lightweight jQuery plugin used to display a user's Instagram photos.",
+  "keywords": [
+    "instagram",
+    "photos",
+    "api",
+    "feed"
+  ],
+  "license": "MIT",
+  "author": "Michael Lynch <mikethomaslynch@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/michael-lynch/instagram-lite"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/michael-lynch/instagram-lite.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "instragramLite*"
```

Comment:
Hey @sufuf3, I just did the two changes that you requested.
I got curious about how would I run the auto-updater locally. I tried what [you suggested before](https://github.com/cdnjs/cdnjs/pull/11583#pullrequestreview-51640690) but I guess that this will just replace the files inside the cdnjs repo from the repo that I specify in the cli argument and not simulate the behavior that we were looking for.
Thanks